easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -498,22 +503,20 @@ def include_defined_parameters(keyset):
                 printed = False
                 for key in group:
                     val = self._config[key][0]
-                    def_val = DEFAULT_CONFIG[key]
-                    if val not in def_val:
+                    if val != default_values[key]:
```

Comment:
replace `val` with `self[key]`, should work (see the `__getitem__` defined in the `EasyConfig` class)